apache2/apache2_util.c

```diff
@@ -344,6 +344,14 @@ static void send_waf_log(const char* timestamp, struct waf_lock* lock, apr_file_
     get_short_filename(waf_filename);
     get_ruleset_type_version(waf_ruleset_info, waf_ruleset_type, waf_ruleset_version); 
 
+    // Format UTC time
+    time_t rawtime;
+    struct tm * timeinfo;
+    char timestamp[30];
+    time(&rawtime);
```

Comment:
You can simplify this to just
`time_t rawtime = time()`

and remove the declaration of an uninitialized variable few lines above

Reply:
Good idea. It was a lazy copy-paste from [cplusplus.com](http://www.cplusplus.com/reference/ctime/strftime/) .

Reply:
Are you sure? When I try I get this:
```
error: too few arguments to function 'time'
     time_t rawtime = time();
                      ^
```

Reply:
Indeed. It should be `time_t rawtime = time(NULL);` then: 
http://man7.org/linux/man-pages/man2/time.2.html